extensions/info/runtime/src/main/java/io/quarkus/info/runtime/InfoRecorder.java

```diff
@@ -143,6 +145,10 @@ public InfoHandler(Map<String, Object> buildTimeInfo, List<InfoContributor> know
                 // also, do we want to merge information or simply replace like we are doing here?
                 finalBuildInfo.put(contributor.name(), contributor.data());
             }
+            Arc.container().listAll(InfoContributor.class)
```

Comment:
Just a small nitpick: We generally avoid streams and lambdas in runtime code.

Reply:
Yes I know, it's just really deep into my hands now, so I tend to forget ;)